site/content/blog/2021-12-01-whats-new-in-svelte-december-2021.md

```diff
@@ -0,0 +1,91 @@
+---
+title: What's new in Svelte: December 2021
+description: "Svelte Summit Fall 2021 Recap, Rich Harris joins Vercel, SvelteKit opens GitHub Discussions"
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+With SvelteKit getting more and more stable each day, there's not much to cover in terms of code changes other than bug fixes... So, in this month's newsletter, we'll be covering Svelte Summit Fall 2021!
+
+If you want to dive deep into the last month's worth of bug fixes, check out the [Svelte](https://github.com/sveltejs/svelte/blob/master/CHANGELOG.md) and [SvelteKit](https://github.com/sveltejs/kit/blob/master/packages/kit/CHANGELOG.md) changelogs, respectively.
+
+
+## What happened at Svelte Summit?
+
+If you missed Svelte Summit, you can watch the entire live stream on [YouTube](https://www.youtube.com/watch?v=1Df-9EKvZr0) and catch a recap in the [#svelte-summit channel on Discord](https://discord.gg/YmHcdnhu).
+
+Here are the highlights:
+- [Rich Harris](https://twitter.com/rich_harris) took us through a tour of Svelte's history and announced [his move to Vercel](https://vercel.com/blog/vercel-welcomes-rich-harris-creator-of-svelte) - where he will be helping maintain Svelte full-time!
+- [Steph Dietz](https://twitter.com/steph_dietz_) explained how Svelte's simple abstractions makes it easy for beginners and experts alike to learn and use JavaScript - without the boilerplate
+- [Kevin Bridges](https://twitter.com/kevinast) dove deep into Svelte's reactivity logic by visualizing it through `ReflectiveCounter` and showing how to "fine tune" it, as needed
+- [Mateo Morris](https://twitter.com/_mateomorris) launched Primo an all-in-one SvelteKit CMS to help build and manage static sites
+- [Guillermo Rauch](https://vercel.com/about/rauchg) explained Vercel's commitment to Svelte, what it means to have Rich on the team, and what's coming next from the company...
+- [Geoff Rich](https://twitter.com/geoffrich_) introduced various ways to modify motion and transitions within Svelte to be more accessible to all users of the web
+- [Dean Fogarty](https://df.id.au/) demoed a number of different use-cases for custom stores - transforming data to and from storage mechanisms within Svelte
+- [Kellen Mace](https://twitter.com/kellenmace) shared how we can let content creators keep using WordPress, while leveraging Svelte on the frontend to provide a phenomenal user experience
+- [Ben Holmes](https://twitter.com/bholmesdev) explained the "islands" architecture and how 11ty + Slinkity can bring these islands to any HTML template
+- [Scott Tolinski](https://twitter.com/stolinski) shared the lessons learned from rewriting LevelUpTutorials in Svelte (from React) and "found developer bliss"
```

Comment:
"from React **to Svelte**" sound better, than "**in Svelte**".

Reply:
"from rewriting LevelUpTutorials in Svelte (from React)" -> "from rewriting React LevelUpTutorials in Svelte"